pkg/controller/restore_controller.go

```diff
@@ -141,7 +142,7 @@ func NewRestoreController(
 
 	c.syncHandler = c.processRestore
 
-	restoreInformer.Informer().AddEventHandler(
+	backupInformer.Informer().AddEventHandler(
```

Comment:
FYI this shouldn't have changed -- you should revert.  May cause challenges when testing as-is :)

Reply:
Yes, it was a find/replace snaffu. Thanks for catching! 🤦‍♀️